### PR TITLE
Add info for Windows users

### LINKED ISSUE #4: Add info about git installing on Windows
Add info about git installing on Windows from http://git-scm.com/download/win.

Otherwise `git clone https://github.com/dab/bemup-workshop-vagrant.git` will not work

### PR DESCRIPTION
**Pull request by alexbaumgertner**
fixes #4 #5 #6 #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ Provisions a clean Ubuntu 12.04 32-bit server instance with all needed Node.js d
 ### Install VirtualBox:
 [https://www.virtualbox.org/wiki/Downloads](https://www.virtualbox.org/wiki/Downloads)
 
+## For Windows users
+Windows is not built into ssh client, you should install, for example [putty](http://www.putty.org/).
```
Comment (dab) [inline README.md]: Windows has no built-in ssh client. You should install and use 3rd party one. For instance [putty]...

File: README.md [hunk 2]
```diff
@@ -12,6 +12,16 @@ Provisions a clean Ubuntu 12.04 32-bit server instance with all needed Node.js d
...
+
+This is the solution for fixing error `The box failed to unpackage properly`:
```
Comment (dab) [inline README.md]: Here the fix for error `...

File: README.md [hunk 3]
```diff
@@ -12,6 +12,16 @@ Provisions a clean Ubuntu 12.04 32-bit server instance with all needed Node.js d
...
+download the `basic_bsdtar.exe` from here [bsdtar](http://sourceforge.net/projects/mingw/files/MinGW/Extension/bsdtar/) and rename it to `bsdtar.exe` and place it in C:\HashiCorp\Vagrant\embedded\gnuwin32\bin\.
```
Comment (dab) [inline README.md]: 2 раза and

File: README.md (hunk 1 above)
Comment (tadatuta) [inline README.md]: предлагаю рекомендовать `git bash`

File: README.md (hunk 2 above)
Comment (tadatuta) [inline README.md]: Here _is_ the fix

File: README.md (hunk 1 above)
Comment (alexbaumgertner) [inline README.md]: @tadatuta оно http://msysgit.github.io?

**Comment by alexbaumgertner** [comment]
@tadatuta @dab :up:

File: README.md [hunk 4]
```diff
@@ -12,6 +12,16 @@ Provisions a clean Ubuntu 12.04 32-bit server instance with all needed Node.js d
 ### Install VirtualBox:
 [https://www.virtualbox.org/wiki/Downloads](https://www.virtualbox.org/wiki/Downloads)
 
+## For Windows users
+Windows has no built-in ssh client. You should install and use 3rd party one. For instance [git bash](http://msysgit.github.io).
+
+Here is the fix for error `The box failed to unpackage properly`:
+download the `basic_bsdtar.exe` from here [bsdtar](http://sourceforge.net/projects/mingw/files/MinGW/Extension/bsdtar/) then rename it to `bsdtar.exe` and place it in C:\HashiCorp\Vagrant\embedded\gnuwin32\bin\.
+
+You can find git client for Windows here: [git for win](http://git-scm.com/download/win).
```
Comment (dab) [inline README.md:21]: here should be git-bash, cause it is `Git for Windows`

**Pull request closed without merging**